pkgs/linux-vf2.nix

```diff
@@ -9,8 +9,9 @@ buildLinux (args // {
   src = fetchFromGitHub {
     owner = "starfive-tech";
     repo = "linux";
-    rev = "JH7110_VisionFive2_devel";
-    sha256 = "sha256-o1k1UDUXUsRb4200zZ5ozVL15g7wsheet8O5UhX2HWY=";
+    ref = "JH7110_VisionFive2_devel";
```

Comment:
That leads to an ```called with unexpected argument 'ref'``` error.

```diff
-   ref = "JH7110_VisionFive2_devel";
+   # following ref = "JH7110_VisionFive2_devel"
```

Could this be converted to a flake input?

Reply:
oh right, fetchgit needs ref, not fetchFromGitHub